layout/layout.go

```diff
@@ -747,3 +766,40 @@ func imageRefAnnotation(additionalNames ...string) map[string]string {
 	}
 	return annotations
 }
+
+// validMediaTypes returns true if media types present in the manifest are the ones defined by the OCI spec
+// Docker Media Types will return false.
+func validMediaTypes(manifest *v1.Manifest) bool {
+	return manifest.MediaType == types.OCIManifestSchema1 &&
+		manifest.Config.MediaType == types.OCIConfigJSON
+}
+
+// overridesMediaType will create a new v1.Image from the provided base image, but replacing
+// manifest media type, config media type and layers media type by the ones defined by the OCI spec
+func overridesMediaType(base v1.Image) v1.Image {
+	config, _ := base.ConfigFile()
```

Comment:
Should we be checking errors here?

Reply:
Same as my above comment, I will change the method signature to throw the error.